WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerce.xcscheme

```diff
@@ -123,6 +123,10 @@
             argument = "mocked-network-layer"
             isEnabled = "NO">
          </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-com.apple.CoreData.ConcurrencyDebug 1"
+            isEnabled = "YES">
```

Comment:
Just a small nit: I think you left the `ConcurrencyDebug` check enabled here 🙌

Reply:
@pmusolino I intentionally enabled that so that as default for now. This would only affect debug build runs. Could you share why we should avoid this?